module/item/systems/repair.js

```diff
@@ -0,0 +1,301 @@
+import { extendedRoll } from "../../scripts/rolls/extendedRoll.js";
+import { RollConfig } from "../../scripts/rollConfig.js";
+import { updateItem } from "../../scripts/item/updateItem.js";
+
+const repairableItemTypes = ['weapon', 'armor']
+const durabilityLocations = [
+    { label: ['WITCHER.Repair.damagedLocations.weapon'], reliability: 'reliable', maxReliability: 'maxReliability' },
+    { label: ['WITCHER.Armor.LocationHead'], reliability: 'headStopping', maxReliability: 'headMaxStopping' },
+    { label: ['WITCHER.Armor.LocationTorso'], reliability: 'torsoStopping', maxReliability: 'torsoMaxStopping' },
+    { label: ['WITCHER.Location.Left', 'WITCHER.Armor.LocationArm'], reliability: 'leftArmStopping', maxReliability: 'leftArmMaxStopping' },
+    { label: ['WITCHER.Location.Right', 'WITCHER.Armor.LocationArm'], reliability: 'rightArmStopping', maxReliability: 'rightArmMaxStopping' },
+    { label: ['WITCHER.Location.Left', 'WITCHER.Armor.LocationLeg'], reliability: 'leftLegStopping', maxReliability: 'leftLegMaxStopping' },
+    { label: ['WITCHER.Location.Right', 'WITCHER.Armor.LocationLeg'], reliability: 'rightLegStopping', maxReliability: 'rightLegMaxStopping' },
+    { label: ['WITCHER.Repair.damagedLocations.shield'], reliability: 'reliability', maxReliability: 'reliabilityMax' },
+]
+
+const repairModifier = 5
```

Comment:
the modifier should be -5 making it implict at - in the formula might lead to confusion in the long term